readme.md

```diff
@@ -1,3 +1,70 @@
 # instructions for all assignments and exercises
 
-Instructions will be released as they are developed
+Instructions will be released as they are developed.
+
+## Assignment Submission Summary
+Assignments are submitted in two parts:
```

Comment:
Assignment submission requires 3 steps:
1. you **push** your changes to GitHub, then
2. you **verify** your changes are now on GitHub, then
3. you **copy/paste** the URL of your GitHub repo into a D2L text box